libs/datalayout/build.gradle

```diff
@@ -5,7 +5,7 @@ plugins {
 }
 
 ext {
-    icebergVersion = '1.2.0'
+    icebergVersion = '1.2.0.4'
```

Comment:
should we consider having a single place for maintaining these versions?

Reply:
Discussed offline. Ideally, it is a good idea to define the dependency versions in a single file(like gradle-versions.properties) and use it across all the modules for consistency. We can pick this up as follow-up change.